nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/ops/transforms/Transforms.java

```diff
@@ -993,6 +993,33 @@ public static INDArray identity(INDArray ndArray, boolean dup) {
         return Nd4j.getExecutioner().exec(dup ? new Identity(ndArray, ndArray.ulike()) : new Identity(ndArray, ndArray))[0];
     }
 
+    /**
+     * Copy
+     * @param ndArray
+     * @return
+     */
+    public static INDArray copy(INDArray ndArray) {
+        return Nd4j.getExecutioner().exec(new Identity(ndArray, ndArray.ulike()))[0];
+    }
+
+    /**
+     * CopyHost
+     * @param ndArray
+     * @return
+     */
+    public static INDArray copyHost(INDArray ndArray) {
+        return Nd4j.getExecutioner().exec(new Identity(ndArray, ndArray.ulike()))[0];
+    }
+
+    /**
+     * DeepCopy
+     * @param ndArray
+     * @return
+     */
+    public static INDArray copyDeep(INDArray ndArray) {
+        return Nd4j.getExecutioner().exec(new Identity(ndArray, ndArray.ulike()))[0];
+    }
```

Comment:
We don't need these methods, for 2 reasons
(a) Transforms class is going away, we'll use codegen to generate the ND4J equivalents of SDMath, SDNN, etc
(b) Copy, CopyHost and CopyDeep are needed for TF import only - I don't want them in the SameDiff or ND4J public API like this... users can use INDArray.dup() to get exactly this